copy/copy.go

```diff
@@ -546,6 +551,25 @@ func (c *copier) copyOneImage(ctx context.Context, policyContext *signature.Poli
 		return nil, "", "", errors.Wrapf(err, "Error initializing image from source %s", transports.ImageName(c.rawSource.Reference()))
 	}
 
+	options.DestinationManifestCheck = true // TODO: Forced enablement for initial PR testing
+	if options.DestinationManifestCheck && options.SignBy == "" {
+		destImageSource, err := c.dest.Reference().NewImageSource(ctx, options.DestinationCtx)
+		if err == nil {
+			retManifest, retManifestType, err = destImageSource.GetManifest(ctx, nil)
```

Comment:
This should specify `targetInstance` so that it benefits multi-arch images as well (and correctly handles updates to a tag between single-arch/multi-arch images).